phoenix-core/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -184,9 +184,9 @@ public static ColumnResolver getResolverForCreation(final CreateTableStatement s
             // A tenant-specific connection may not create a mapped VIEW.
             if (connection.getTenantId() == null && statement.getTableType() == PTableType.VIEW) {
                 ConnectionQueryServices services = connection.getQueryServices();
-                byte[] fullTableName = SchemaUtil.getPhysicalName(
-                        SchemaUtil.getTableNameAsBytes(baseTable.getSchemaName(), baseTable.getTableName()),
-                        connection.getQueryServices().getProps()).getName();
+                boolean isNamespaceMapped = SchemaUtil.isNamespaceMappingEnabled(statement.getTableType(), connection.getQueryServices().getProps());
+                byte[] fullTableName = SchemaUtil.getPhysicalHBaseTableName(
```

Comment:
As @ankitsinghal mentioned in the previous comment, I have changed this to handle "AAA.BBB"(without schema) correctly using SchemaUtil.getPhysicalHBaseTableName() (which I've further modified in this commit).